packages/patterns/src/types.ts

```diff
@@ -72,11 +73,11 @@ export type { FullCompare } from '@endo/marshal';
  *    contain more than one key.
  */
 export type Key = Exclude<
```

Comment:
For both before and after, what is `Exclude` doing that the limited parameterization of `Passable` is not doing?

Reply:
Yeah I'm not sure, but I'd like to avoid more drive by changes.

Reply:
ok. resolving